lib/openstudio-standards/standards/ashrae_90_1_prm/ashrae_90_1_prm.ChillerElectricEIR.rb

```diff
@@ -14,16 +14,17 @@ def chiller_electric_eir_apply_efficiency_and_curves(chiller_electric_eir)
 
     # Set the efficiency value
     cop = chiller_electric_eir_standard_minimum_full_load_efficiency(chiller_electric_eir)
-    kw_per_ton = cop_to_kw_per_ton(cop)
     if cop.nil?
```

Comment:
rubocop is suggesting guard clause format here:

unless cop
  # log stuff
  return false
end